Makefile

```diff
@@ -46,7 +45,7 @@ tc-worker-env:
 	docker build -t taskcluster/tc-worker-env -f tc-worker-env.Dockerfile .
 
 tc-worker:
-	CGO_ENABLED=$(CGO_ENABLE) GOARCH=amd64 go build
+	CGO_ENABLED=$(CGO_ENABLED) GOARCH=amd64 go build
 	docker build -t taskcluster/tc-worker -f tc-worker.Dockerfile .
```

Comment:
hmm, this is a little unpleasant... if the system has a newer glibc version than what is inside the docker container, then we get issues...

Maybe we fix that issue, if we ever hit it... I'm guessing the solution is to use a different docker image to build the it...